README.md

```diff
@@ -354,7 +315,7 @@ integrated with coveralls and travis-ci.org:
   ```json
   {
      "script": {
-       "test": "nyc tap ./test/*.js",
+       "test": "nyc mocha",
```

Comment:
I made this change because `tap` uses `nyc` under the hood for coverage, so it doesn't make sense to go this way. Most of our tutorials use `mocha` so I thought I'd just continue that trend.